docs/ref/pgcopydb_config.rst

```diff
@@ -78,6 +78,8 @@ configuration then the sections ``exclude-schema`` and ``exclude-table`` are
 disallowed. We would not know how to handle tables that exist on the source
 database and are not part of any filter.
 
+NOTE: Materialized views are also considered as tables during the filtering.
+
```

Comment:
I think this only makes sense if the existing `[exclude-table]` section allows filtering-out materialized views. Is that the case? (can't remember now and being lazy).

Reply:
@dimitri , originally it wasn't supported by pgcopydb, but this PR adds the capability to filter matview using [exclude-table].